src/transaction/log_manager.c

```diff
@@ -7923,6 +7930,8 @@ log_rollback (THREAD_ENTRY * thread_p, LOG_TDES * tdes, const LOG_LSA * upto_lsa
 	    case LOG_END_OF_LOG:
 	    case LOG_SMALLER_LOGREC_TYPE:
 	    case LOG_LARGER_LOGREC_TYPE:
+	    case LOG_START_ATOMIC_REPL:
```

Comment:
Don't add a fatal error when rollback hits a start/end atomic log record. No action is needed here; move them to the same group as `LOG_SUPPLEMENTAL_INFO`.